db/migrate/20240311173819_remove_profiles_and_add_parent_from_adopter_foster_profiles.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveProfilesAndAddParentFromAdopterFosterProfiles < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :adopter_foster_profiles, :adopter_account_id, :bigint
+    remove_column :adopter_foster_profiles, :foster_account_id, :bigint
```

Comment:
is this removal needed in this particular branch? I had it because I was on your branch where you had already added this column in a migration.

Reply:
I added this back in

Reply:
Ah. Why is that?